File: pyocel/ocel/exporter/exporter.py
from ocel.exporter import oceljson, ocelxml


def apply(log, output_path, parameters=None):
    if ".json" in output_path:
        return oceljson.apply(log, output_path, parameters=parameters)
    else:
        return ocelxml.apply(log, output_path, parameters=parameters)

File: ocel/src/main/java/it/unibz/inf/pm/ocel/exporter/Exporter.java
package it.unibz.inf.pm.ocel.exporter;

import org.dom4j.DocumentException;

import java.util.Map;

public class Exporter {
    public static void apply(Object log, String output_path,String ... parameters) throws DocumentException {
        if (output_path.indexOf(".json") != -1){
            Oceljson.apply(log,output_path);
        }else{
            Ocelxml.apply((Map)log, output_path);
        }
    }
}
